docs/references/strategies/languages/python/python.md

```diff
@@ -5,7 +5,7 @@ The python buildtool ecosystem consists of three major toolchains: setuptools
 
 | Strategy                                       | Direct Deps        | Transitive Deps    | Edges              | Container Scanning |
 | ---------------------------------------------- | ------------------ | ------------------ | ------------------ | ------------------ |
-| [pipenv](pipenv.md)                            | :heavy_check_mark: | :heavy_check_mark: | :heavy_check_mark: | :x:                |
+| [pipenv](pipenv.md)                            | :heavy_check_mark: | :x:                | :heavy_check_mark: | :x:                |
```

Comment:
[Prefer]  In this table, we should include something other than :x:, for transitive deps, as we are still including transitives in the final analysis, and generated reports from such analysis

Reply:
The effect we're describing is what "edges" in this table indicates. We should just update that.

```suggestion
| [pipenv](pipenv.md)                            | :heavy_check_mark: | :heavy_check_mark: | :x:                | :x:                |
```